test/sharness/lib/test-lib.sh

```diff
@@ -276,3 +276,22 @@ test_curl_resp_http_code() {
 	cat curl_output
 	return 1
 }
+
+test_must_be_empty() {
+	if test -s "$1"
+	then
+		echo "'$1' is not empty, it contains:"
+		cat "$1"
+		return 1
+	fi
+}
+
+test_should_contain() {
+	test "$#" = 2 || error "bug in the test script: not 2 parameters to test_should_contain"
```

Comment:
I decided to mimic original errors: examples are [here](https://github.com/git/git/blob/master/t/test-lib-functions.sh#L355) and [here](https://github.com/git/git/blob/master/t/test-lib-functions.sh#L687).